src/managers/GuildChannelManager.js

```diff
@@ -9,7 +9,8 @@ const PermissionOverwrites = require('../structures/PermissionOverwrites');
 const ThreadChannel = require('../structures/ThreadChannel');
 const { ChannelTypes, ThreadChannelTypes } = require('../util/Constants');
 
-let cacheWarningEmitted = false;
+let cacheWarningEmitted = false,
+  storeChannelDeprecationEmitted = false;
```

Comment:
Please do not do this.
```suggestion
let cacheWarningEmitted = false;
let storeChannelDeprecationEmitted = false;
```

Reply:
Sure but may I ask why?

Reply:
Because we don't do that nasty stuff around here.